client/state/reader-ui/reducer.js

```diff
@@ -76,7 +76,32 @@ const getInitialLastActionState = () => {
 /*
  * Holds the last action that requires the user to be logged in
  */
-export const lastActionRequiresLogin = ( state, action ) => {
+export const lastActionRequiresLogin = ( state = null, action ) => {
```

Comment:
This should be all the same as before, but without the getInitialLastActionState initialization.